schemas/+aod/trackMotion3D.m

```diff
@@ -0,0 +1,86 @@
+
+function [x,y,z,t] = trackMotion3D(fn, analysischan) 
+
+%% test function computeshifts
+% requires z:\libraries\matlab in path
+run(getLocalPath('/lab/libraries/hdf5matlab/setPath.m')) ;
+
+downsampleFactor = 1 ; % skip frames if > 1
+numFramesToAverageForRef = 10 ; % frames to average to determine the reference frame against which motion is computed
+useMultiCore = true ;
+postRefStartFrameTime = 0 ; % N secs to skip before analyzing motion signals, in many cases, right after starting the point scan, there is a slight drift
+
+br = aodReader(fn, 'Motion') ;
```

Comment:
here I think this should use getFile(fn, 'Motion') to convert  the path from /raw/blah to the real location